.github/workflows/ci.yml

```diff
@@ -379,7 +378,10 @@ jobs:
                   else
                     echo "No report will be created for '$GITHUB_EVENT_NAME' event"
                   fi
-                
+
+            - name: 'Add annotation to workflow run linking to PR'
+              run: |
+                  echo "::notice::🔗🔗 The e2e report for this run is available at https://woocommerce.github.io/woocommerce-test-reports/pr/${{ github.event.pull_request.number }}/e2e"
```

Comment:
For events other than `pull_request` this will add an invalid url. Maybe only running this step for pull_request would be better?